src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -75,14 +76,16 @@ describe 'ProductTypeImporter', ->
     # increase timeout so we will have time to delete all previous product types
     this.timeout 60000
 
-    sphereClient.productTypes
-    .perPage(50)
-    .process (res) ->
-      console.log "Deleting old product types", res.body.results.length
-      Promise.map res.body.results, (productType) ->
-        sphereClient.productTypes.byId(productType.id)
-        .delete(productType.version)
-      , concurrency: 10
+    Helper.deleteAllProducts sphereClient
```

Comment:
You could also move this function to the helper file so you export only this one and it calls the others implicitly